tasks/Task.cpp

```diff
@@ -135,6 +140,29 @@ void Task::processIO()
         _timestamp_estimator_status.write(mTimestampEstimatorStatus);
     }
 }
+Eigen::Vector3d Task::filterAngularVelocity(Eigen::Vector3d const& angular_velocity)
+{
+    int filter_size = _angular_velocity_filter_size.get();
+    if (filter_size <= 1) {
```

Comment:
the angular_velocity_filter_size is the property we use to define the size in samples of the filter, but if the size of the filter is bigger than 1, wouldn't this bypass the filter altogether?

Reply:
This tests if the size is smaller or equal than 1, not bigger